pkgs/applications/audio/psst/default.nix

```
@@ -0,0 +1,30 @@
+{ lib, rustPlatform, fetchFromGitHub, pkg-config, dbus, fetchgit, ... }:
+
+let
+  rev = "318e069ca85d0f58358d9bc8d879ecdd4d4ec0ea";
+in
+
+rustPlatform.buildRustPackage rec {
+  pname = "psst";
+  version = builtins.substring 0 7 rev;
+
+  src = fetchgit {
+    url = "https://github.com/jpochyla/${pname}";
```

```suggestion
    url = "https://github.com/jpochyla/psst";
```
Stop substituting every occurrences of pname and use fetchFromGitHub.